backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -98,7 +98,23 @@ public void testGetTokensForShop_success() throws Exception {
                 inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).size());
     }
 
-    private UUID addShopToRepository() {
+    @Test
+    public void testGetShop_success() {
+        // Arrange.
+        UUID shopId = addShopToRepository(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
+        
+        // Act.
+        Shop newShop = inMemoryVirtualQueueRepository.getShop(shopId);
+
+        // Assert.
+        assertEquals("Shop Owner Id ", SHOP_OWNER_ID, newShop.getShopOwnerId());
+        assertEquals("Shop Name ", SHOP_NAME, newShop.getShopName());
+        assertEquals("Shop Address ", SHOP_ADDRESS, newShop.getAddress());
+        assertEquals("Shop Phone Number ", PHONE_NUMBER, newShop.getPhoneNumber());
+        assertEquals("Shop Type ", SHOP_TYPE, newShop.getShopType());
+    }
+
+    private UUID addShopToRepository(String SHOP_OWNER_ID, String SHOP_NAME, String SHOP_ADDRESS, String PHONE_NUMBER, String SHOP_TYPE) {
         Shop shop = new Shop(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
```

Comment:
Why use these when the values are being passed as params to this method?

Reply:
These will change when the variable names in the params of this method changes

Reply:
Changed.